src/hypervisor.c

```diff
@@ -107,6 +108,43 @@ cc_oci_expand_net_device_cmdline(struct cc_oci_config *config, guint index) {
 	return g_strdup("");
 }
 
+/*
+ *
+ *
+ */
+#define DPDK_CMDLINE_OBJ "memory-backend-file,id=dpdkmem,size=2048M,mem-path=/dev/hugepages,share=on,prealloc=on"
```

Comment:
@jodh-intel I like the idea of using @VM_MEM@ and @VM_MEM_MAX@ and specifying in the code. This will be first step in making them dynamic.